repl/evaluator.go

```diff
@@ -675,6 +762,162 @@ func (e *Evaluator) Status() string {
 	return status.String()
 }
 
+func (e *Evaluator) YAMLConfig() string {
+	var status strings.Builder
+	status.WriteString("# CEL environment YAML\n")
+	conf, err := e.getEffectiveEnv().ToConfig("repl-session")
+	if err != nil {
+		panic(fmt.Errorf("REPL env broken %w", err))
```

Comment:
Could we consider a more graceful error handling here, unless if you believe this is an unrecoverable situation? Largely thinking in terms of preventing app restarts on the web based REPL

Reply:
Done. I don't think this is realistic to trigger but propagating the error is probably cleaner.